services-manager/src/main/resources/application-eureka.yaml

```diff
@@ -17,3 +17,31 @@
 eureka:
   client:
     enabled: true
+

```

Comment:
Unfortunately, this will probably cause problems when trying to start the palisade service with `eureka,h2` (or `eureka,jvm`

Reply:
Lists may not get amended with overlapping profiles, just straight overwritten

Reply:
Confirmed, lists are not amended they are overwritten. Reverted back.